composer/packages/theme/src/definitions/views/api-designer.less

```diff
@@ -29,22 +29,68 @@
             .action-container {
                 margin-bottom: 1em;
             }
-
-            .accordion.ui.fluid.styled {
-                padding-bottom: .5em !important;
+            .accordion {
                 margin-top: 20px;
                 border-left: 0;
                 border-right: 0;
                 border-top: 1px solid #e8e8e8;
                 border-bottom: 1px solid #e8e8e8;
-                .inline-editor {
-                    font-weight: 500;
-                    &:hover {
-                        opacity: 0.5;
+                box-shadow: none;
+                .path-op-container {
+                    display: inline;
+                    float: right;
+                    .get {
+                        border: 1px solid #2980b9;
+                        color: #2980b9;
+                        font-weight: 400;
+                    }
+                    .put {
+                        border: 1px solid #fca130;
+                        color: #fca130;
+                        font-weight: 400;
+                    }
+                    .post {
+                        border: 1px solid #27ae60;
+                        color: #27ae60;
+                        font-weight: 400;
+                    }
+                    .delete {
+                        border: 1px solid #f93e3e;
+                        color: #f93e3e;
+                        font-weight: 400;
+                    }
+                    .patch {
+                        border: 1px solid #8e44ad;
+                        color: #8e44ad;
+                        font-weight: 400;
+                    }
+                    .head {
+                        border: 1px solid #3F51B5;
+                        color: #3F51B5;
+                        font-weight: 400;
+                    }
+                    .options {
+                        border: 1px solid #43A047;
+                        color: #43A047;
                         font-weight: 400;
-                        border: 0;
                     }
                 }
+                .accordion{
+                    margin-top: 0px;
+                    .title{
```

Comment:
Keep a new line in between child class name and ending of parent class properties, for a better readability and standard.

Reply:
Will be tracked with issue #13334